### ISSUE TITLE
Basic ideas

### ISSUE THREAD

**Issue by Science-Recon** [PhelanBavaria/ancienttimeline#55]
So, having looked through the ideas and playing the game for a bit, I think we should redo the basic ideas. Most don't really fit with the time, and while they could be renamed and such as some have, it'd probably be easier and better to just redo them. In addition, we could look at policies, though I'm not sure what to do for them. ET uses different ideasets for early eras, so I suppose we could do something like that and maybe have some more dynamic ones as well (available to specific cultures, religions, locations, techs, times &c.) What do you think?

**Comment by Firesoul7** [comment]
Sounds good

**Comment by PanzerfaustJoe** [comment]
I started I guess. I already changed Humanist Ideas to Altruistic ideas.

**Comment by Science-Recon** [comment]
So, these are some of the ones I've got so far. Haven't got all of them done yet, and the numbers aren't final.

[00_basic_ideas.txt](https://github.com/PhelanBavaria/ancienttimeline/files/785853/00_basic_ideas.txt)

**Comment by Science-Recon** [comment]
Right, I've finished a few idea groups. I'll localise them in a bit and I'll add some more. Do we want to add the 'early ideas' from ET and/or do we want to create an equivalent 'really early' group of ideasets for the beginning of AT?
